src/fuzzer.rs

```diff
@@ -124,17 +129,24 @@ pub fn fuzz() -> Result<()> {
         *api.clone(),
     )?;
 
-    // A minimization+queue policy to get testcasess from the corpus
-    let scheduler = IndexesLenTimeMinimizerScheduler::new(PowerQueueScheduler::new(
-        &mut state,
-        &coverage_observer,
-        PowerSchedule::FAST,
-    ));
+    let combined_map_observer =
+        combined_observer(&mut endpoint_coverage_client, code_coverage_client.as_mut())
+            .track_indices();
+
+    // A minimization+queue policy to get testcases from the corpus
+    let scheduler = IndexesLenTimeMinimizerScheduler::new(
+        &combined_map_observer,
+        PowerQueueScheduler::new(&mut state, &combined_map_observer, PowerSchedule::FAST),
+    );
 
     // A fuzzer with feedbacks and a corpus scheduler
     let mut fuzzer = StdFuzzer::new(scheduler, collective_feedback, objective);
 
-    let collective_observer = tuple_list!(endpoint_observer, coverage_observer, time_observer);
+    let collective_observer = tuple_list!(
```

Comment:
I am not sure what you exactly tried to achieve. But The cause of the missing key lies here, I believe.

The `combined_map_observer` is not present in this list, which is later used to get the `all_maps` entry from.

Reply:
Nope, this is a question for @grebnetiew :)

Reply:
I have attempted to fix the problem by (inlining the creation of the multimap observer because naming its type is prohibitively time consuming, and) using the multimap observer also in the tuple list. Does this fix the missing key problem you speak of @ThomasTNO ?

Reply:
This is the behaviour I observe
```
thread 'main' panicked at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl-0.13.2/src/stages/calibrate.rs:156:34:
Could not find entry matching Handle { name: "code_coverage", type: "libafl::observers::map::ExplicitTracking<libafl::observers::map::StdMapObserver<u8, false>, true, true>" }
stack backtrace:
   0: rust_begin_unwind
             at /rustc/051478957371ee0084a7c0913941d2a8c4757bb9/library/std/src/panicking.rs:652:5
   1: core::panicking::panic_fmt
             at /rustc/051478957371ee0084a7c0913941d2a8c4757bb9/library/core/src/panicking.rs:72:14
   2: <libafl_bolts::tuples::RefIndexable<RM,M> as core::ops::index::Index<&libafl_bolts::tuples::Handle<T>>>::index
             at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl_bolts-0.13.2/src/tuples.rs:638:13
   3: <libafl::stages::calibrate::CalibrationStage<C,E,O,OT> as libafl::stages::Stage<E,EM,Z>>::perform
             at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl-0.13.2/src/stages/calibrate.rs:156:34
   4: libafl::stages::Stage::perform_restartable
             at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl-0.13.2/src/stages/mod.rs:127:13
   5: <(Head,Tail) as libafl::stages::StagesTuple<E,EM,<Head as libafl::state::UsesState>::State,Z>>::perform_all
             at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl-0.13.2/src/stages/mod.rs:214:17
   6: <libafl::fuzzer::StdFuzzer<CS,F,OF> as libafl::fuzzer::Fuzzer<E,EM,ST>>::fuzz_one
             at /home/thomas/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libafl-0.13.2/src/fuzzer/mod.rs:804:9
   7: wuppiefuzz::fuzzer::fuzz
             at ./src/fuzzer.rs:334:9
   8: wuppiefuzz::main
             at ./src/main.rs:87:34
   9: core::ops::function::FnOnce::call_once
             at /rustc/051478957371ee0084a7c0913941d2a8c4757bb9/library/core/src/ops/function.rs:250:5
```

We are now missing a different map ;)

Reply:
Possibly we are misinterpretating the working of the `MultiMapOberserver`..

a94cb25 seems to make it all operational.

Reply:
Currently JaCoCo coverage seems to work fine. Blackbox mode is operational. LCOV coverage seems to fail (tested on Python). I suspect an existing issue though. Possibly the coverage agent is too fast / slow compared to the coverage gathering resulting in an empty map.